Engine/source/platform/platformNet.cpp

```diff
@@ -1262,7 +1233,7 @@ void Net::process()
          {
             // try to connect
             out_h_addr.port = currentSock->remotePort;
-            const sockaddr *ai_addr = NULL;
+            const sockaddr* ai_addr = NULL;
```

Comment:
Oh no. This is improper as it is the variable that is a pointer, not a pointer type. Very bad habit when declaring variables.
The original code was proper.

Reply:
The same argument can be made for it being a pointer type which is exactly the case at my previous job. There needs to be a programming style guide to agree upon for cases like this.

Reply:
It has to do with the language. This applies for both C and C++:
```cpp
const sockaddr* a1, b2, c3;
```

`a1` is a pointer variable to `const sockaddr` memory, while `b2` and `c3` are `const sockaddr` variables.
Which is why it is improper to have the asterisk next to the data type because it is not apart of the data type declaration.

Reply:
the dumbness of C and C++. :x